playground/nextjs/pages/orgswitcher/[[...index]].tsx

```diff
@@ -0,0 +1,13 @@
+import { OrganizationSwitcher } from '@clerk/nextjs';
+import type { NextPage } from 'next';
+import React from 'react';
+
+const OrganizationSwitcherPage: NextPage = () => {
+  return (
+    <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+      <OrganizationSwitcher />
+    </div>
+  );
+};
+
+export default OrganizationSwitcherPage;
```

Comment:
Let's add this in the navbar instead.
I already have this ready locally, so, please remove the playground changes so we can merge. 

Lastly, in the future, let's split commits depending on the changes they introduce, for example, in this case we'd have a commit for the refactor and a second commit for the playground changes - that way I'd just cherry pick the changes we need without bothering you again